content_creator.py

```diff
@@ -150,30 +213,77 @@ def main(circle_artifacts):
-                print('skipping {}'.format(package))
-            else:
-                merge_script_package_to_yml(package, package_dir, BUNDLE_POST)
+        # handles nested package directories
+        create_unifieds_and_copy(package_dir)
 
     for content_dir in CONTENT_DIRS:
         print(f'Copying dir {content_dir} to bundles...')
         copy_dir_files(content_dir, BUNDLE_POST)
 
     copy_test_files(BUNDLE_TEST)
 
+    # handle copying packs content to content_new.zip and content_test.zip
+    packs = get_child_directories(PACKS_DIR)
+    for pack in packs:
+        # each pack directory has it's own content subdirs, 'Integrations', 'Scripts', 'TestPlaybooks', 'Layouts' etc.
+        sub_dirs_paths = get_child_directories(pack)
+        for sub_dir_path in sub_dirs_paths:
+            dir_name = os.path.basename(sub_dir_path)
+            if dir_name == 'TestPlaybooks':
+                copy_test_files(BUNDLE_TEST, sub_dir_path)
+            else:
+                # handle one-level deep content
+                copy_dir_files(sub_dir_path, BUNDLE_POST)
+                if dir_name in DIR_TO_PREFIX.keys():
+                    # then it's a directory with nested packages that need to be handled
+                    # handle nested packages
+                    create_unifieds_and_copy(sub_dir_path)
+
+    # handle copying packs content to packs_bundle for zipping to `content_packs.zip`
+    for pack in packs:
+        pack_name = os.path.basename(pack)
+        pack_dst = os.path.join(PACKS_BUNDLE, pack_name)
+        os.mkdir(pack_dst)
+        pack_dirs = get_child_directories(pack)
+        pack_files = get_child_files(pack)
+        # copy first level pack files over
+        for file_path in pack_files:
+            shutil.copy(file_path, os.path.join(pack_dst, os.path.basename(file_path)))
+        # handle content directories in the pack
+        for content_dir in pack_dirs:
+            dir_name = os.path.basename(content_dir)
+            dest_dir = os.path.join(pack_dst, dir_name)
+            os.mkdir(dest_dir)
+            if dir_name in DIR_TO_PREFIX.keys():
+                packages_dirs = get_child_directories(content_dir)
+                for package_dir in packages_dirs:
+                    package_dir_name = os.path.basename(package_dir)
+                    dest_package_dir = os.path.join(dest_dir, package_dir_name)
+                    os.mkdir(dest_package_dir)
+                    package_dir_with_slash = package_dir + '/'
+                    merge_script_package_to_yml(package_dir_with_slash, dir_name, dest_path=dest_package_dir)
+
+                    # also copy CHANGELOG markdown files over
+                    package_files = get_child_files(package_dir)
+                    changelog_files = [
+                        file_path
+                        for file_path in package_files if ('CHANGELOG' in file_path and file_path.endswith('.md'))
```

Comment:
all change logs ends with `CHANGELOG.md`
```suggestion
                        for file_path in package_files if 'CHANGELOG.md' in file_path
```